device-server/src/main/kotlin/com/badoo/automation/deviceserver/ios/simulator/data/Media.kt

```diff
@@ -27,6 +28,21 @@ class Media(
         restartAssetsd()
     }
 
+    fun addMedia(fileName: String, data: ByteArray) {
+        val tmpFile = File.createTempFile(null, fileName)
+        tmpFile.writeBytes(data)
+
+        val addCmd = listOf(
+           "xcrun", "simctl", "addmedia", udid, tmpFile.absolutePath
+        )
```

Comment:
does it work with remote simulators?

Reply:
It should work since we are calling it suing IRemote interface. The rest of xcrun commands are using it as well.